solver/src/solver/http_solver/model.rs

```diff
@@ -26,14 +26,22 @@ pub struct OrderModel {
 }
 
 #[derive(Debug, Serialize)]
-pub struct UniswapModel {
-    pub token1: String,
-    pub token2: String,
-    #[serde(with = "serde_with::rust::display_fromstr")]
-    pub balance1: u128,
-    #[serde(with = "serde_with::rust::display_fromstr")]
-    pub balance2: u128,
-    #[serde(with = "serde_with::rust::display_fromstr")]
+pub struct PoolTokenData {
```

Comment:
I think we should add a test in this file so that the JSON format is documented in code.

Reply:
Yes good idea, Although I might prefer to add this in the following PR that changes the tokens to H160. 

Note that the solver team has agreed to this proposed change along with the Token addresses. See here:

https://github.com/gnosis/gp-v2-solver/issues/147#issuecomment-865689927